src/main/java/org/broadinstitute/hellbender/tools/sv/SVFeature.java

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import htsjdk.tribble.Feature;
+
+import java.util.List;
+
+public interface SVFeature extends Feature {
+    SVFeature extractSamples( final List<String> sampleNames, final Object header );
```

Comment:
The collection of sampleNames to extract could be a Collection rather than a List.  (I'll make that change.) It's always used by iterating over it.  No advantage to making it a Set, and has the disadvantage that it doesn't allow you to specify the order in which you'd like the sample data written for DepthEvidence.

Reply:
Oops.  Wrong.  You're right that we usually call the contains method.  Reworking.